### DISCUSSION TITLE
Do we need num of cores to be equal num threads, or can we use hyper-threading ?

### DISCUSSION THREAD

**Discussion by ch3ll0v3k** [Chia-Network/chia-blockchain#2420] [General]
Hi!

Do we need num of cores to be equal num threads, or can we use hyper-threading ?
for creating new plots.

Thx

**Comment by cccat6** [comment]
Currently, only phrse 1 can use the benefit of multi-threads (Saying in the official document). You can assume plotting is a single-thread program and give each task one thread (not core). Give 2 threads for phrse 1 (`-r 2`) is fair enough, more doesn't make a significant difference.

**Reply by ch3ll0v3k** [reply]
thank you!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
